src/components/SourceTabs.js

```diff
@@ -114,13 +116,27 @@ const SourceTabs = React.createClass({
       click: () => closeTabs(tabs)
     };
 
-    showMenu(e, buildMenu([
+    const showSourceMenuItem = {
```

Comment:
I landed this work earlier today (I think). A rebase would be healthy either way :)